ui-core/src/styles/inputs/statusMessage.css

```diff
@@ -31,3 +31,60 @@
     }
   }
 }
+
+.status-message-wrapper-tooltip {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  width: 320px;
+  top: 74px;
+  gap: 12px;
+  border-radius: 8px;
+  box-shadow:
+    0 10px 20px 0 rgba(0, 0, 0, 0.2),
+    0 1px 0 rgba(255, 255, 255, 0.6) inset;
+  padding: 6px 12px 10px;
+  min-height: 64px;
+  z-index: 10;
+
+  &.small {
+    top: 58px;
+  }
+
+  &.error {
+    background-color: theme('colors.error.5');
+  }
+
+  &.info {
+    background-color: theme('colors.info.5');
+  }
+
+  &.warning {
+    background-color: theme('colors.warning.5');
+  }
+
+  &.tooltip-left {
+    left: -5px;
+  }
+
+  &.tooltip-right {
+    right: -5px;
+  }
```

Comment:
The offset is from the text field edges 

![Capture d’écran 2024-12-20 à 14 39 10](https://github.com/user-attachments/assets/2fc304c2-0756-43ba-99e8-12284b512b5a)

```suggestion
  &.tooltip-left {
    left: 8px;
  }

  &.tooltip-right {
    right: 8px;
  }
```